int64 ID
float64 signal_strength
